README.md

```diff
@@ -36,44 +40,71 @@ The instructor will then review the pull request and make comments should furthe
 - Question (IL): What is a "libref", and how does it differ from a "LIBNAME"?  In particular, what fundamental distinction causes one to be written out in lower-case letters and the other in upper-case letters?
 - Answer (SG): A libname statement is the syntactical statement used to initiate a particular library. The libref is the actual syntax used to name it.
 
+-Question (YM): What is the difference between Temporary Data Library and Permanent Data Library?
```

Comment:
The format for bullet points is as follows:

- [Question|Answer] ([initials]): [text]

Note the spaces after the hyphen and [Question|Answer].